src/hardware/vga_dac.cpp

```diff
@@ -22,7 +22,7 @@
 
 #include "inout.h"
 #include "mem.h"
-#include "reelmagic/vga_passthrough.h"
+#include "reelmagic.h"
```

Comment:
Ok so this commit is doing a lot more than just removing a few unused declarations. You could either call it "Various cleanup changes" but better yet, you could split the unrelated changes, which should be easy to do.

Reply:
Well, the commit's title must reflect what the commit does 😄 So either change the commit message so it's a lot more broad (worse solution), or split the commit and make the commit messages of the resulting commits more specifi (better solution) 😄